test/ensembl/ensembl.fetchAndLoadById.test.js

```diff
@@ -0,0 +1,174 @@
+const fs = require('fs');
+const jwt = require('jsonwebtoken');
+
+const { fetchAndLoadById } = require('../../src/ensembl');
+const { ApiConnection } = require('../../src/graphkb');
+const { request } = require('../../src/util');
+
+// Mock the request function from utility module in order to avoid real http requests
+jest.mock('../../src/util.js', () => {
```

Comment:
rather than mocking this whole module could you use the node-fetch mock similar to https://github.com/bcgsc/pori_graphkb_loader/pull/75 and then simply pass a mock object for the `conn` (ApiConnection class)

Reply:
this will have the added advantage of avoiding global